tw-gaffer-jta-starter/src/main/java/com/transferwise/common/gaffer/starter/DefaultEnvironmentValidator.java

```diff
@@ -1,5 +1,6 @@
 package com.transferwise.common.gaffer.starter;
 
+import com.transferwise.common.gaffer.GafferJtaProperties;
```

Comment:
Unused import.

Reply:
Ah, right. Didn't notice the class move to other package which created this import.